### DISCUSSION TITLE
How do we get a current edit out of `draft` status?

### DISCUSSION THREAD

**Discussion by travisrr** [amzn/selling-partner-api-models#3677] [General]
Hello!

We recently needed to edit the callback URL for our application and it is stuck in `draft` mode.  Does anyone know how to get it out of draft mode in this case?  Seems like were missing a simple step here.

To be clear, the current app is published and has been for some time, we have an `edit` in process (call back URL change) and the edit is now in the `draft` status and we need to get it processed.

Any help would be amazing thanks in advance!

**Comment by rugved1991** [comment]
Hi @travisrr,

You can initiate the App listing process again by clicking on "Edit Listing" in the dropdown in front of the application. Once you publish the app, the new redirect URL will be reflected in the OAuth flow. 

Best,
Rugved
Solutions Architect, SP-API

**Reply by travisrr** [reply]
thx @rugved1991 !  I found the area.  i dont think its very intuitive, but at least there is a process.  

now currently, all of our sellers who are already OAuth'd are listing products just fine and the connection to SP-API is present but because we changed the callback URL, all of our new sellers signing up each day cant auth their Seller Central account with us.

Do you know how long this approval process takes for an app that's already been in production and approved for some time now?  Can you help us expedite it in any way?  Really appreciate any help you can give as it's costing us thousands of dollars each day...

**Reply by travisrr** [reply]
Support Case ID if this helps sir: ID 13667544391

**Reply by Landsailor** [reply]
@travisrr 

We are stuck in the same boat after adding Seller - Product Listings permissions to the Developer Role (which was approved immediately) and the app. Support keeps sending us the the same canned response.

Do you have any hints as to any magic words to get them to move it out of draft status?

Thanks!
